cli/internal/publish/images/images.go

```diff
@@ -39,6 +41,25 @@ type listKey struct {
 	name, sum string
 }
 
+func QuickContentType(filename string) (string, error) {
```

Comment:
How about the existing name `DetectContentType`? There's no verb on this name.

Reply:
Fixed in https://github.com/cloudquery/cloudquery/pull/19373/commits/269d066b6501c4059bb52924177756c35ad5bbb9